krux/logging.py

```diff
@@ -120,9 +121,12 @@ def syslog_setup(name, syslog_facility, **kwargs):
     logger = logging.getLogger(name)
     # On Linux, Python defaults to logging to localhost:514; on Ubuntu, rsyslog is not configured
     # to listen on the network. On other platforms (Darwin/OS X at least), Python by default sends
-    # to syslog vi a method by which syslog is listening.
-    if platform.system() == 'Linux':
-        handler = logging.handlers.SysLogHandler('/dev/log', facility=syslog_facility)
+    # to syslog via a method by which syslog is listening. Also need to test for the existence of the
+    # device, it apparently does not exist in a docker container; at the very least, not in a Travis CI
+    # build docker container. Can't use os.path.isfile() which returns False for devices.
+    log_device = '/dev/log'
+    if platform.system() == 'Linux' and os.path.exists(log_device):
```

Comment:
You probably also want to check if it is writeable with `os.access`: https://docs.python.org/2.6/library/os.html#os.access